webapp/src/webapp/connections/views/connection_list.cljs

```diff
@@ -79,6 +79,30 @@
          (when (= selected-resource value)
            [:> Check {:size 16}])])]]]])
 
+(defn can-connect? [connection]
+  (not (and (= "disabled" (:access_mode_runbooks connection))
+            (= "disabled" (:access_mode_exec connection))
+            (= "disabled" (:access_mode_connect connection)))))
+
+(defn can-open-web-terminal? [connection]
+  (if-not (#{"tcp" "httpproxy"} (:subtype connection))
+
+    (if (or (= "enabled" (:access_mode_runbooks connection))
+            (= "enabled" (:access_mode_exec connection)))
+      true
+      false)
+
+    false))
+
+(defn can-open-native-client? [connection]
+  (if (#{"database" "application"} (:type connection))
+
+    (if (= "enabled" (:access_mode_connect connection))
+      true
+      false)
+
+    false))
```

Comment:
Hey @rogefm we have native SSH which currently doesn't allow access to the web console.
I try to maintain the list of current features updated as best as I can in the Connection [Quick Start documentation](https://hoop.dev/docs/quickstart/ssh). It could be a good place to consult before implementing these conditions.

Another option that we could do is allowing the server to inform the available connection types and not rely in this static information.